examples/remote_data_manager.py

```diff
@@ -0,0 +1,56 @@
+"""
+========================================
+Using sunpy.data.manager
+========================================
+
+This is an example to show how to use remote data manager to
+handle remote data in sunpy.
+"""
+# Remote data manager is used to handle the usage of remote files in the code
+# base in a sane way.
+##############################################################################
+from sunpy.data import manager
+
+##############################################################################
+# Let's start by defining a function which uses some remote files.
+# Let's assume the function requires a file, say http://212.183.159.230/5MB.zip, which has
+# the SHA1 hash '0cc897be1958c0f44371a8ff3dddbc092ff530d0'.
+
+@manager.require('test_file',
+                 ['http://212.183.159.230/5MB.zip'],
+                 '0cc897be1958c0f44371a8ff3dddbc092ff530d0')
+def test_function():
+    pass
+
+
+##############################################################################
+# To access the downloaded file inside the function, you can use `manager.get` function
```

Comment:
Does this not need to be the full import path for sphinx to find it?

Reply:
Maybe use double ticks? Because it's an object and not a class. Can sphinx do objects too?

Reply:
Single ticks are enough but what i don't know if Spinx will find the method without the full import path.